treeherder/perf/stats.py

```diff
@@ -510,6 +541,33 @@ def interpret_silverman_kde(base_data, new_data, lower_is_better):
                     "ci_warning": ci_warning,
                 }
                 modes.append(mode_info)
+        else:
+            for i, interval in enumerate(base_intervals):
+                tup = interval
+                if len(tup) != 2:
+                    return None, None, None, None, None, None
+
+                start, end = tup
+                shift = 0
+                ci_low = 0
+                ci_high = 0
```

Comment:
Given that these are default values, you can set the fields to these values directly instead of making intermediate variables. Same is true for the values below.